From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 29 Aug 2022 14:21:44 -0400
Subject: [PATCH] get_password_passthrough(): handle the callback context right

Right now, we have a few callback functions for PK11_Authenticate(), and
they take different arguments.  This is incorrect; none of the callers
ever pass anything through except our CMS context.

This fixes get_password_passthrough() to correctly accept the CMS
context and get the passthrough data from cms->pwdata instead of trying
to treat the CMS context as the pwdata.

Related: rhbz#2122777

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/password.c | 16 +++++++++++++---
 1 file changed, 13 insertions(+), 3 deletions(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
